README.md

```diff
@@ -1,28 +1,70 @@
+[![Build Status](https://travis-ci.com/rednavis/spring-webflux-swagger-demo.svg?branch=master)](https://travis-ci.com/rednavis/spring-webflux-swagger-demo)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/143579ed76ba41e88c2d0a1d8744a48d)](https://app.codacy.com/gh/rednavis/spring-webflux-swagger-demo?utm_source=github.com&utm_medium=referral&utm_content=rednavis/spring-webflux-swagger-demo&utm_campaign=Badge_Grade_Dashboard)
+[![codecov](https://codecov.io/gh/rednavis/spring-webflux-swagger-demo/branch/master/graph/badge.svg)](https://codecov.io/gh/rednavis/spring-webflux-swagger-demo)
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+
 # spring-graphql-microservice
+Demo of using Spring, GraphQL and microservices.
+
+## Prerequisites
+- OpenJDK 12.0.2 or higher
+- Gradle 6.1.1 or higher
+
+## How to build
+```
+$ ./gradlew clean bootJar jibDockerBuild
+```
 
-## Build
-1. `./gradlew clean bootJar jibDockerBuild`
-2. `docker-compose up` 
+## How to lauch
+After build run 
+```
+$ docker-compose up
+```
 
-## Eureka
+## Heroku setup
+* Login to your heroku account from CLI
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/gh/rednavis/spring-graphql-microservice/pullRequest?prid=5023967)